code/index.html

```diff
@@ -48,17 +85,39 @@ <h1>Does the person have</h1>
       <main class="game-board" id="board"></main>
     </section>
 
+    
+    
+    
+    
     <section id="winOrLose" class="win-or-lose-wrapper">
       <div class="win-or-lose">
         <img
           class="guess-who-icon"
-          src="images/guess-who-bubble.png"
+          src="images/reed.svg"
           alt="Guess Who"
         />
         <h1 id="winOrLoseText"></h1>
         <button id="playAgain" class="filled-button">PLAY AGAIN</button>
       </div>
     </section>
-    <script src="script.js"></script>
+
+    
+    
+    <!---- When you click on Guess-button ---->
+    <section id="guessConfirmation" class="confirm-guess-wrapper">
+      <div class="confirm-guess">
+        <h1 id="confirmGuessText"></h1>
+
+        
+      </div>
+    </section>
+
+
+   
+   
+   
+   
```

Comment:
Do you remember about the prettier code formatter extension they mentioned in the resources for this week? It is an extension that helps clean up your code, so you don't have for example unnecessary spaces and wrong indentation.